cmake/projects/libsodium/hunter.cmake

```diff
@@ -17,10 +18,14 @@ hunter_add_version(
     00a999629f1f8f500cef29818135f8f889f44e41
 )
 
+hunter_configuration_types(libsodium CONFIGURATION_TYPES Release)
 if(MSVC)
     hunter_pick_scheme(DEFAULT url_sha1_libsodium_msbuild)
 else()
     hunter_pick_scheme(DEFAULT url_sha1_libsodium_autogen_autotools)
```

Comment:
I was not able to use `make VERBOSE=1` to print the full compilation command line during the build for some reason, even with `make VERBOSE=1` I get the terse compilation output as something along the lines of `CC dir/file.c`